next-desolates-map/components/SettingsPage/SettingsPage.tsx

```diff
@@ -0,0 +1,466 @@
+import { useContext, useState } from "react";
+import CameraMovement from "../../core/modules/CameraMovement";
+import { Clusters, SpaceRendererContext, Stages } from "../../pages/_app";
+import { searchForPlanetAndChangeStage } from "../RenderController/RenderController";
+import Image from "next/image"
+import pic from "../../public/profile.png"
+
+
+export default function SettingsPage() {
+    const {
+        changeStage,
+        p5,
+        cam,
+        setOngoingCamMov,
+        celestialObjects,
+        setCluster,
+        setSelectedPlanet,
+    } = useContext(SpaceRendererContext);
```

Comment:
We can remove all of these, they are not helpful here I think

Reply:
Done.